citesphere/src/main/java/edu/asu/diging/citesphere/web/user/ItemController.java

```diff
@@ -33,6 +37,21 @@ public String getItem(Authentication authentication, Model model, @PathVariable(
             List<String> fields = new ArrayList<>();
             citationManager.getItemTypeFields((IUser)authentication.getPrincipal(), citation.getItemType()).forEach(f -> fields.add(f.getFilename()));
             model.addAttribute("fields", fields);
+            if(index != null) {
+                model.addAttribute("index", index);
+                Map<String,String> results = citationManager.getPrevAndNextCitation((IUser)authentication.getPrincipal(), zoteroGroupId, collectionId, page, sortBy, Integer.valueOf(index));
+                if(results.containsKey("next")) {
+                    model.addAttribute("next", results.get("next"));
+                    model.addAttribute("nextIndex", results.get("nextIndex"));
+                }
+                if(results.containsKey("previous")) {
+                    model.addAttribute("previous", results.get("previous"));
+                    model.addAttribute("prevIndex", results.get("prevIndex"));
+                }
```

Comment:
hard-coded strings are always bad, either create a class for this or use an enum.

Reply:
this refers to the results map